mlflow/sparkml.py

```diff
@@ -0,0 +1,122 @@
+"""
```

Comment:
What do you think about just calling this module `spark.py`? Nobody else really uses the name `sparkml` and there's only one ML library in Spark.

Reply:
Yeah makes sense. I will rename it.